khal/khalendar/event.py

```diff
@@ -556,6 +568,33 @@ def _alarm_str(self) -> str:
             alarmstr = ''
         return alarmstr
 
+    @property
+    def _status_str(self) -> str:
+        if self.status == 'CANCELLED':
+            statusstr = ' ' + self.symbol_strings['cancelled']
```

Comment:
I'd like to make a case against adding a leading space, or at least to make this configurable. Such a leading space should really be part of the format specification, though I realise it's currently not possible to work with conditionals there.

However, `{status: <1}` will probably suit anyone better when the status symbol is to be used mid-field, whereas adding all the statuses to the end of the line won't make a difference anyway.

Reply:
fixed